packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -304,6 +305,14 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     return null;
   }
 
+  private _onRenderDescription = (props: ITextFieldProps): JSX.Element | null => {
+    const { description } = props;
+    if (description) {
+      return (<span className={ css('ms-TextField-description', styles.description) }>{ description }</span>);
+    }
+    return null;
```

Comment:
The following code will do the same

return description && <span....>

Reply:
Slight challenge here was that we'd have to add string to the return type, because of falsey empty strings.